src/app/page.tsx

```diff
@@ -1,15 +1,7 @@
-import { createServerComponentClient } from '@supabase/auth-helpers-nextjs'
-import { cookies } from 'next/headers'
 import { css } from '../../styled-system/css'
 import Link from 'next/link'
 
 export default async function LandingPage() {
-  const supabase = createServerComponentClient({ cookies })
-
-  const {
-    data: { session },
-  } = await supabase.auth.getSession()
-
   return (
     <div
       className={css({
```

Comment:
todo/bug: This page always shows "Login" (line 20) even if we are already logged. It's missing a conditional there

Reply:
Just added it :sweat_smile:

Reply:
fixed